aws/required-permissions.md

```diff
@@ -0,0 +1,101 @@
+# Permissions required by the templates
```

Comment:
We have three different scenarios need permissions. 

- a. the permission to deploy templates. 
- b. the permissions that deployed resources will have. 
- c. the management role permissions that backend needs. 

I thought this document covers only #b? if so, I suggest replacing "templates" with "deployed stacks"

```suggestion
# Permissions required by the deployed stacks
```

Reply:
It includes B and C.

Reply:
And d. the permissions of custom resource, which will fail the deployment if the permissions is not granted.

Reply:
I see... If #A is not included, I still think "deployed stacks" is better than "templates".

Reply:
Per offline discussion, we will add a note in the description instead of changing title.